.taskcluster.yml

```diff
@@ -123,99 +123,126 @@ tasks:
             owner: ${user}@users.noreply.github.com
             source: ${repository}/raw/${head_rev}/.taskcluster.yml
       in:
-        - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "synchronize"]'
-          then:
-            $let:
-              pull_request_title: ${event.pull_request.title}
-              pull_request_number: ${event.pull_request.number}
-              pull_request_url: ${event.pull_request.html_url}
-            in:
+        $flatten:
```

Comment:
Question: Is this `$flatten` call to make sure `.taskcluster.yml` just outputs a single task?

Reply:
It's because I have [nested `$if`](https://github.com/mozilla-mobile/fenix/pull/2341/files#diff-ac0229d1171c0983b27003af4c7441a5R172), and `json-e` forces each `$if` to be an array element.

So, if one of the github-release statements matches, the result is:
```
-
  -
    payload: ...
    ...
```
The `$flatten` makes it:
```
-
  payload: ...
  ...
```